newrelic_lambda_cli/integrations.py

```diff
@@ -525,10 +525,11 @@ def install_log_ingestion(
                 failure("Failed to create 'newrelic-log-ingestion' function: %s" % e)
                 return False
         else:
-            failure(
+            warning(
                 "CloudFormation Stack NewRelicLogIngestion exists (status: %s), but "
```

Comment:
I'm not sure this is the real solution. The case where there's a stack and not a function is actually an error. But what I think you really want is to skip the stack check/creation entirely if the user is using the lambda extension.

Reply:
@kolanos  - Not gonna get to this tonight! But yeah, I think that's a great idea to implement the logic to audit lambda extension usage and exit "early" vs later, so skipping the stack check/creation.  I'll hammer on it soon tho! 👍 🚀 😄